commands/invoke.go

```diff
@@ -63,32 +71,32 @@ var invokeCmd = &cobra.Command{
 }
 
 func runInvoke(cmd *cobra.Command, args []string) error {
-	var services stack.Services
-
 	if len(args) < 1 {
 		return fmt.Errorf("please provide a name for the function")
 	}
+	functionName = args[0]
 
 	if missingSignFlag(sigHeader, key) {
 		return fmt.Errorf("signing requires both --sign <header-value> and --key <key-value>")
 	}
 
-	var yamlGateway string
-	functionName = args[0]
+	err := validateHTTPMethod(httpMethod)
+	if err != nil {
+		return nil
+	}
 
-	if len(yamlFile) > 0 {
```

Comment:
Why is this no longer required? I can't see equivalent code on the right hand side.

Reply:
We were only using the stack file to read the provider url. After switching to the go-sdk the default client is retrieved by calling `GetDefaultSDKClient()`. This code should take care of initializing the client with the correct gateway url.

However after checking this a noticed this was not the case and published a fix.